user/newsflash/template.py

```diff
@@ -0,0 +1,53 @@
+pkgname = "newsflash"
+pkgver = "3.3.4"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "blueprint-compiler",
+    "cargo-auditable",
+    "desktop-file-utils",
+    "gettext",
+    "meson",
+    "ninja",
+    "pkgconf",
+]
+makedepends = [
+    "clapper-devel",
+    "libadwaita-devel",
+    "libxml2-devel",
+    "openssl-devel",
+    "rust-std",
+    "webkitgtk4-devel",
+]
+pkgdesc = "Feed reader designed for the GNOME desktop"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-3.0-or-later"
+url = "https://gitlab.com/news-flash/news_flash_gtk"
+source = f"{url}/-/archive/v.{pkgver}/news_flash_gtk-v.{pkgver}.tar.gz"
+sha256 = "f408f4c2d1e1507008ef583868b8482708d13269b86b8e22d2ba73da9c93a0ae"
+
+
+def post_patch(self):
+    from cbuild.util import cargo
+
+    cargo.Cargo(self, wrksrc=".").vendor()
+
+
+def init_build(self):
+    from cbuild.util import cargo
+
+    renv = cargo.get_environment(self)
+    self.make_env.update(renv)
+
+
+def pre_build(self):
+    self.mkdir(self.cwd / "build/src/release", parents=True)
+    (self.cwd / "build/src/release/news_flash_gtk").touch(0o755)
+
+
+def post_install(self):
+    self.uninstall("usr/bin/io.gitlab.news_flash.NewsFlash")
+    self.install_bin(
+        f"build/src/{self.profile().triplet}/release/news_flash_gtk",
+        name="io.gitlab.news_flash.NewsFlash",
+    )
```

Comment:
look at how loupe does it

Reply:
oh. my. god? thats why i didnt understand what they were doing differently before! they literally just patch the god damn meson.build! sorry for the hacks, thats much cleaner

Reply:
yea, and don't forget to remove the CARGO_HOME part of the env thing in src/meson.build too (pretty much all the cargo+meson projects are the same)